src/main/java/Bob.java

```diff
@@ -0,0 +1,260 @@
+            } catch (NoCommandFormatException e) {
+                printNoCommandFormat(commandType);
+            }
+        }
+    }
+
+    private static void printNumberFormatException() {
+        String exceptionMessage = LINE_STRING +
+                " 😥 Number format exception encountered! Input may be corrupted\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    private static void printIOException() {
+        String exceptionMessage = LINE_STRING +
+                " 😥 IO issue encountered! Unable to read file\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    private static void printInputMismatch() {
+        String exceptionMessage = LINE_STRING +
+                " 😥 There is some issue with getting you input\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    private static void printNoSuchMethod() {
+        String exceptionMessage = LINE_STRING +
+                " 😥 I don't quite get what the command means\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    private static void printNoCommandLabel(Command commandType) {
+        String commandName = commandType.name().toLowerCase();
+        String exceptionMessage = LINE_STRING +
+                " 😥 You gotta tell me what is the task for " + commandName + "\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    private static void printNoCommandFormat(Command commandType) {
+        String commandName = commandType.name().toLowerCase();
+        String exceptionMessage = LINE_STRING +
+                " 😥 You gotta use the time marker for " + commandName + "\n" +
+                "and the time which it happens\n" +
+                LINE_STRING;
+        System.out.println(exceptionMessage);
+    }
+
+    /**
+     * String before first space taken as command rest as args
+     */
+    private static boolean scanSwitch(String inputString, Command commandType) throws NoSuchMethodException, NoCommandLabelException, NoCommandFormatException {
+        String[] inputStringArr = inputString.split(" ");
+        boolean isScanning = true;
+
+        switch (commandType) {
+        case LIST:
+            taskList.printList();
+            break;
+        case DONE:
+            taskList.completeTask(inputStringArr[1]);
+            break;
+        case DELETE:
+            taskList.deleteTask(inputStringArr[1]);
+            break;
+        case TODO:
+            taskList.addTodo(inputString);
+            break;
+        case DEADLINE:
+            taskList.addDeadline(inputString);
+            break;
+        case EVENT:
+            taskList.addEvent(inputString);
+            break;
+        case BYE:
+            isScanning = false;
+            break;
+        default:
+            throw new NoSuchMethodException();
+        }
+        return isScanning;
```

Comment:
Good job here, in keeping the code [simple](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#practice-kissing) and easy to read.